src/pages/verify/e-ids/danish-mitid-erhverv.mdx

```diff
@@ -58,6 +58,20 @@ import JwtPayloadViewer from '../../../components/JwtPayloadViewer';
 
 The `sub`, `nameidentifier` and `uuid` values here will not be the same as for a corresponding [citizen-MitID login](/verify/e-ids/danish-mitid/#jwttoken-examples-mitid-citizen), even if the user chose to use their personal MitID for login.
 
+## Collecting CPR numbers (optional)
+
+Business users (both _employees_ and _company signatories_) may not always have CPR numbers associated with their profiles.
+
+CPR numbers are also not required for accessing information about the business entities where the user holds a position as a signatory or an employee. Criipto relies on the user's UUID to retrieve this information from the MitID Erhverv API. Therefore, if you don't need CPR numbers in business logins, no additional configuration is required.
```

Comment:
A small twist to this: If the end-customer has not yet migrated to MitID Erhverv, only company-signatory-lookups-via-CPR will allow business users to log in.

I am not sure about what will happen to these customers when NemID closes, but we may want to add a note about this (maybe) final "transitional period" ?